src/main/java/org/jabref/model/openoffice/uno/DocumentAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.model.openoffice.uno;
+
+import com.sun.star.text.XTextDocument;
+import com.sun.star.text.XTextRange;
+
+/**
+ * Represents a document annotation.
+ *
+ * @param doc    The document
+ * @param name   For the bookmark.
+ * @param range  Cursor marking the location or range for the bookmark.
```

Comment:
Some comments say: `If an XTextSection by that name already exists, LibreOffice (6.4.6.2) creates a section with a name different from what we requested, in "Section {number}" format.`. Maybe, that should be put here, too? Or to the method getting this new object as parameter? - In all cases, the comment of the parameter should not get lost.